src/components/auth/SocialAuthButton.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const { REACT_APP_GOOGLE_LOGIN, REACT_APP_FACEBOOK_LOGIN, REACT_APP_TWITTER_LOGIN } = process.env;
+const socialAuthButton = () => (
+  <div className="social-media-Login">
+    <div>
+      <div className="social-auth-text">
+        <span className="valign-wrapper">______________</span>
+          OR LOGIN WITH
+        <span className="valign-wrapper">______________</span>
+      </div>
+    </div>
+    <div className="icon-container">
+      <div>
+        <a className="icon-link google-icon" href={REACT_APP_GOOGLE_LOGIN}>
+          <i className="fab fa-google " />
+          &nbsp; google
```

Comment:
Please, we don't add `&nbsp`. Remove all instances added and add styles accordingly.

Reply:
resolved